src/chess/position.h

```diff
@@ -65,6 +67,8 @@ class Position {
   const ChessBoard& GetBoard() const { return us_board_; }
   // Gets board from the point of view of opponent.
   const ChessBoard& GetThemBoard() const { return them_board_; }
+  // Gets board from the point of view of the white player.
+  const ChessBoard& GetWhiteBoard() const { return us_board_.flipped() ? them_board_ : us_board_; };
```

Comment:
Have clang-format installed as a plug-in to visual studio. Must say that the default behavior is not always ideal, that is, I find the lines a little bit too short. For example: I do not use the clang-format on the position_test.cc file because it breaks up the fen-strings!